### ISSUE TITLE
Reinstallation problem

### ISSUE THREAD

**Issue by Regdan** [quantumsheep/sshs#50]
Hi everyone,

I don't know why, today when i launched "SSHS", i had this error : 
2023/01/30 14:37:15 strconv.Atoi: parsing "22\t": invalid syntax

So after some tries, i uninstall sshs application in order to reinstall it with chocolatey and i don't know why, it says me continually that the sshs-windows-386.exe program is using by another process.

![image](https://user-images.githubusercontent.com/6512898/215539527-40d0aeb9-5e58-4c11-9ba7-bd9c214e9ad6.png)

Can you help me with thoses issues ?

**Comment by quantumsheep** [comment]
Hey ! Seems like you have a `sshs` process running on your computer. Have you tried killing the running process or rebooting your computer before reinstalling?

**Comment by Regdan** [comment]
Yes, i tried to reboot my computer because i don't find the running process :( . And it seems the "NuGetScratch" folder only appear when i try to install sshs :( 
It's as if the installer try to launch the sshs-windows-386.exe twice.

**Comment by quantumsheep** [comment]
Hey @jakublevy 👋  Do you have an idea?

**Comment by jakublevy** [comment]
I am not really sure what is happening.
I've tested the package (reinstalling, upgrading) and cannot replicate the error.

Try
1. kill all sshs processes by executing `Get-Process -Name 'sshs*' | % { Stop-Process $_ -Force }`
2. delete `"$env:temp\chocolatey"`, `"$env:ChocolateyInstall\lib\sshs"` and `"$env:ChocolateyInstall\bin\sshs.exe"`
3. install sshs again

**Comment by Regdan** [comment]
Ok so, i've executed the command but nothing has been display. I've deleted all files in the specified locations.

Then there is the installation :

![image](https://user-images.githubusercontent.com/6512898/216335560-13764795-44be-4963-bd01-eb7b530817bf.png)

Nothing more :(

**Comment by jakublevy** [comment]
You executed `"$env:temp\chocolatey"`, `"$env:ChocolateyInstall\lib\sshs"` and `"$env:ChocolateyInstall\bin\sshs.exe"` as commands, but I meant to delete those.

You can delete them in powershell by executing
```
Remove-Item -Path "$env:temp\chocolatey" -Force -Recurse
Remove-Item -Path "$env:ChocolateyInstall\lib\sshs" -Force -Recurse
Remove-Item -Path "$env:ChocolateyInstall\bin\sshs.exe" -Force
```

So try again, first execute `Get-Process -Name 'sshs*' | % { Stop-Process $_ -Force }` then execute the three commands above and then try `choco install sshs`

**Comment by Regdan** [comment]
Hello guys,

Thanks for your answers and i don't know why, i tried the installation this morning and that worked.... I haven't tried the last answer of @jakublevy. Now sshs is reinstalled but now i've this error on launching :

![image](https://user-images.githubusercontent.com/6512898/217512483-2bfe133d-ac28-4642-aefc-cc50ed269954.png)

Any ideas ?

Again, thanks for all :)

**Comment by quantumsheep** [comment]
@Regdan Hey! You have a tab space after your port argument in your ssh config
